src/FINDOLOGIC/Api/Responses/Xml21/Properties/Promotion.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Xml21\Properties;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+use SimpleXMLElement;
+
+class Promotion
+{
+    /** @var string $image */
+    private $image;
+
+    /** @var string $link */
+    private $link;
+
+    /**
+     * Promotion constructor.
+     * @param SimpleXMLElement $response
+     */
+    public function __construct($response)
```

Comment:
Please typehint this with `SimpleXMLElement` and drop the PHPDoc as it is not really needed anyway.